_posts/2022-01-11-lxqt-1.1-preview.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: LXQt 1.1.0 Preview
+slug: lxqt-1.1-preview
+date: feb 09, 2022, 19:00
+promoted: true
+categories: blog
+---
+
+ What can be expected in the next LQXt release, coming in spring? Here some highlights.
+ Together with some bugfixes and more translations done we are quite happy to have achieved already quite a few
+ improvements - hopefully there will be some more.
+
+### PCmanFM-qt
+
+* "Last opened files" in menu "File"
+*  Clicking "Show in Folder" from the browser's download menu the downloaded file is already selected.
+
+<br/>
+### QTerminal
+
+* Improved bookmarks: Editing all bookmarks (no hardcoded ones anymore),
+ fixed filtering
+* Important bugfixes for both dropdown terminal and QTerminal
+
+<br/>
+### Panel
+
+* More compact configuration dialog, better for low resolution screens
+* Desktop names are shown in "Move to desktop ..." in taskbar menu
+
+<br/>
+<img src="../../../../../images/posts/panelsettings.png" width="400" />
+
+### Session Settings
+
+* Added an option for global screen scaling
+
+<br/>
+### Notifications Settings
+
+* Improved ordering in settings dialog
+
+<br/>
+<img src="../../../../../images/posts/notification-settings.png" width="400"/>
+
+
+### Qt6 Coming
+
+* Integration of the appearance of applications in Qt6: icons, font, colors and filedialog
+* The port of parts of LXQt to Qt6 is on the way.
+
+<br/>
+### Appearance
+
+* Some default palettes will be preconfigured
+* Improved default "Vanilla" layout
+* A [common directory](https://github.com/lxqt/lxqt-themes/tree/master/wallpapers) for all theme wallpapers with some new wallpapers.
+
+
+<br/>
+<img src="../../../../../images/posts/plasma-logo-dark.png" width="600"/>
+
+
+<br/>
+### xdg-desktop-portal-lxqt
+
+The backend implementation of [xdg-desktop portal](https://github.com/flatpak/xdg-desktop-portal#readme) is ready for releasing! It allows to use the lxqt filedialog in GTK and other applications. See the [wiki](https://github.com/lxqt/lxqt/wiki/ConfigMiscellaneous#lxqt-filedialog-in-gtk-applications) for details.
```

Comment:
I would explicitly say somewhere, that it is only partial implementation of the xdg-desktop-portal. As we really implement only the FileChooser API and non of the others APIs.

Reply:
Could you write this text? You're way more into it.

Reply:
`Backend implementation of [xdg-desktop-portal](https://github.com/flatpak/xdg-desktop-portal#readme) (currently only [org.freedesktop.portal.FileChooser](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.FileChooser) API) is ready for releasing!`

Reply:
Thanks!